app/main.ipynb

```diff
@@ -56,33 +57,57 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
-   "outputs": [],
    "source": [
-    "# write your code here (for Python default syntax)"
+    "arr = [0] * 1000\n",
+    "sys.getsizeof(arr)"
    ],
    "metadata": {
     "collapsed": false,
     "ExecuteTime": {
-     "end_time": "2023-07-03T12:02:41.673421Z",
-     "start_time": "2023-07-03T12:02:41.666277Z"
+     "end_time": "2024-11-07T07:20:55.531626Z",
+     "start_time": "2024-11-07T07:20:55.522789Z"
     }
-   }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "8056"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "execution_count": 2
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
-   "outputs": [],
    "source": [
-    "# write your code here (for numpy syntax)"
+    "arr_np = np.full(1000, 0, dtype=np.int8)\n",
+    "sys.getsizeof(arr_np)"
```

Comment:
The use of `np.full` with `dtype=np.int8` is a good choice to minimize memory usage for the NumPy array. This meets the task requirement to use as little memory as possible.